pkg/api/k8s_versions.go

```diff
@@ -13,6 +13,39 @@ import (
 // k8sComponentVersions defines "best known-working" component versions for each minor version of k8s
 // Don't delete these hash table keys as they are consumed independently by AKS!
 var k8sComponentVersions = map[string]map[string]string{
+	"1.15": {
+		"dockerEngine":                     "1.13.*",
+		"dashboard":                        "kubernetes-dashboard-amd64:v1.10.1",
+		"exechealthz":                      "exechealthz-amd64:1.2",
+		"addon-resizer":                    "addon-resizer:1.8.4",
+		"heapster":                         "heapster-amd64:v1.5.4",
+		"metrics-server":                   "metrics-server-amd64:v0.3.2",
```

Comment:
`metrics-server` was the only component that was bumped to a new version, according to [the release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#changelog-since-v1140).

Reply:
Reverting to v0.2.1 seems to get `kubectl top nodes` working again, so let's separate concerns and update `metrics-server` elsewhere. See #1172 and #1109.